app/components/candidate_interface/continuous_applications/course_review_component.rb

```diff
@@ -42,6 +44,10 @@ def study_mode_row(application_choice)
           value: application_choice.current_course_option.study_mode.humanize,
         }
       end
+
+      def current_course
+        @current_course ||= @application_choice.current_course
```

Comment:
In my experience its not good to have implicit dependencies in a helper. It is a well known cause of confusion in view helpers with instance variables.

Can we pass in the course to the method rather than depend on the local method?

Reply:
changed with: https://github.com/DFE-Digital/apply-for-teacher-training/pull/9153/commits/4be15010cf63971ae58e494265fe34ed22ddef33